src/site/content/en/blog/vitals-spa-faq/index.md

```diff
@@ -37,6 +37,11 @@ of the technology. While this is not possible in every case today (due to
 limitations in the web platform), we are [actively working on closing those
 gaps](#what-is-google-doing-to-ensure-mpas-do-not-have-an-unfair-advantage-compared-to-spas).
 
+{% Aside %}
+Update: The work is anow available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.
```

Comment:
@malchata do you think we should also include the date that these updates were added, for additional context? For example "Update (February 2023):"

Reply:
You could, but the `updated` value in the front matter has also been provided. I don't think it's necessary, but it wouldn't be a problem if it were added.

Reply:
Looks like it is used in a few other places, but no consistently. But since it does says Update, I'm gonna add it! Seems weird to have an "update" without saying when it was updated now I think about it.